bitnami/shared/certs.yaml

```diff
@@ -1,70 +1,55 @@
 apiVersion: cert-manager.io/v1
-kind: Issuer
-metadata:
-  name: kafka-selfsigned-ca-issuer
-spec:
-  selfSigned: {}
----
-apiVersion: cert-manager.io/v1
-kind: Certificate
-metadata:
-  name: kafka-selfsigned-ca
-spec:
-  issuerRef:
-    kind: Issuer
-    name: kafka-selfsigned-ca-issuer
-  secretName: kafka-selfsigned-ca
-  commonName: kafka-shared
-  isCA: true
----
-apiVersion: cert-manager.io/v1
-kind: Issuer
-metadata:
-  name: kafka-selfsigned-issuer
-spec:
-  ca:
-    secretName: kafka-selfsigned-ca
----
-apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: kafka-shared-0-tls
 spec:
   issuerRef:
-    kind: Issuer
-    name: kafka-selfsigned-issuer
+    kind: ClusterIssuer
+    name: kafka-shared-selfsigned-issuer
   secretName: kafka-shared-0-tls
+  commonName: pubsub/kafka-shared-0.kafka-shared-headless
+  dnsNames:
```

Comment:
Should we add kafka-shared-0.kafka-shared-headless.sys-pubsub.svc.cluster.local as well? That's what the broker pods will use as a host?

Reply:
Is this going to be used for mtls host validation? 
Also, I think we said that the brokers shall advertise provider specific domains (cluster.merit)? In this case it will have to be patched downstream

Reply:
```
org.apache.kafka.common.errors.SslAuthenticationException: SSL handshake failed
Caused by: javax.net.ssl.SSLHandshakeException: No subject alternative DNS name matching kafka-shared-0.kafka-shared-headless.pubsub.svc.cluster.local found.
```
you are correct, it is needed